docs/installation_configuration_guide/ScenarioDeploymentConfiguration.md

```diff
@@ -3,26 +3,32 @@ title: Deployment
 sidebar_position: 3
 ---
 
-# Deployment Manager configuration
+# Scenario Deployment configuration
 
-Deployment Manager deploys scenarios from the Designer to the engine on which scenarios are processed.
-Check [configuration areas](./Common.md#configuration-areas) to understand where Deployment Manager configuration should be
-placed in Nussknacker configuration.
+In order to deploy scenario on the given [Engine](/about/engine), you need to configure the deployment.
 
-Below you can find a snippet of Deployment Manager configuration.
+For the scenario's deployment is responsible Designer's extension called [Deployment Manager](/about/GLOSSARY#deployment-manager).
+To enable given [Deployment Manager](/about/GLOSSARY#deployment-manager) you need to place their jar package in the Designer's classpath.
+Nussknacker is distributed with three default [Deployment Managers](/about/GLOSSARY#deployment-manager) (`flinkStreaming`, `lite-k8s`, `lite-embedded`). Their jars are located in the `managers`
+directory. Parameters available in the deployment configuration, depends on which [Deployment Manager](/about/GLOSSARY#deployment-manager) you've selected.
```

Comment:
```suggestion
directory. Your deployment configuration should match the [Deployment Manager](/about/GLOSSARY#deployment-manager) you've selected.
```
Don't know if that's what you meant. I read it as "If you set type = "flinkStreaming" you configure that specific DM and you should provide values for it specifically"

Reply:
changed